### DISCUSSION TITLE
Device Simulator receives optimization update to dramatically improve Mini Program installation speeds

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#248] [Announcements]
Zepp OS team has been committed to optimizing the developer experience, and we have recently optimized the installation speed of Mini Programs in the Simulator, which has brought significant improvements to the experience.

This update covers the following models Balance, Cheetah Pro, Cheetah (Round), Cheetah (Square), Bip 5, Falcon, T-Rex 2, GTR 4, GTS 4.

We invite developers to install the latest version of the device Simulator to experience it!

<img width="796" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/de5bc7e4-1f35-44e8-bf12-532d687914eb">

**Comment by zdunex25** [comment]
@MyoungXUE cool, but when it comes to gtr4 since v1.0.3 we cannot test Widgets ("path": "secondary-widget/index",) settings item is not showing up in either 1.0.4 and 1.0.5, would be nice if you sort it out.. as for now i have to use at least two system images to test everything
